### DISCUSSION TITLE
"A response with status code 2 was received" hundreds of times in the log file. Please help.

### DISCUSSION THREAD

**Discussion by Atlntssplayer** [DNSCrypt/dnscrypt-proxy#2348] [Q&A]
Hi

I have hundreds of this same line in my log file:
[INFO] A response with status code 2 was received - this is usually a temporary, remote issue with the configuration of the domain name

I have no idea what it means, where it comes from, if it's on my side or remote or anything.
I couldn't find anything online about it either that I could comprehend, I'm not an IT person.
Would be good if dnscrypt-proxy said more about this if possible. Not everybody is a tech person who uses github and this software.
My log level is set to 0, most verbose.

This started happening after I upgraded from an earlier version to version 2.1.4 of dnscrypt-proxy.

I have 7 dns servers configured as "static" in my config, I have been using these servers for years mostly without problems.
Now I'm getting this error in the log file and software on my system became kind of unrealible resolving addresses.

Any info to solve this or to educate me is appreciated. Thank you!

**Comment by lifenjoiner** [comment]
> [INFO] A response with status code 2 was received - this is usually a temporary, remote issue with the configuration of the domain name

The message is clear: **remote** (server) issue and usually temporary on the queried domain names. They say can't give a correct answer.
